src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/GlobalSuppressions.cs

```diff
@@ -13,3 +13,10 @@
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0006:Do not mix attributes from different versions of MEF", Justification = "<Pending>", Scope = "type", Target = "~T:Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands.AbstractAddClassProjectCommand")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0006:Do not mix attributes from different versions of MEF", Justification = "<Pending>", Scope = "type", Target = "~T:Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands.AddClassProjectCSharpCommand")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0006:Do not mix attributes from different versions of MEF", Justification = "<Pending>", Scope = "type", Target = "~T:Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands.AddClassProjectVBCommand")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "RS0006:Do not mix attributes from different versions of MEF", Justification = "<Pending>", Scope = "type", Target = "~T:Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.NuGetDependenciesSubTreeProvider")]
```

Comment:
Replace all attributes in this file with a single attribute that has no scope and target. We should also consider just disabling RS0006 rule in our base [ruleset](https://github.com/dotnet/roslyn-project-system/blob/master/build/Rulesets/Roslyn.ruleset#L65) - change the action to "None", and file an issue to re-enable it once the underlying issue https://github.com/dotnet/roslyn-analyzers/issues/955 is fixed.